bugwarrior/services/gitea.py

```diff
@@ -370,7 +368,7 @@ def __init__(self, *args, **kw):
             'import_labels_as_tags', default=False, to_type=bool
         )
         self.label_template = self.config.get(
-            'label_template', default='{{label}}', to_type=six.text_type
+            'label_template', default='{{label}}', to_type=bool
```

Comment:
`bool` seems wrong. I suspect this whole `self.label_template` definition should probably be eliminated and the whole `get_tags` function could be reduced by using `self.get_tags_from_labels`. See example: https://github.com/GothenburgBitFactory/bugwarrior/blob/71efc97435a442646782dcb0979df43f69ce955d/bugwarrior/services/github.py#L305-L307